src/network/sntp_client.c

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2023 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/logging/log.h>
+#include "sntp_client.h"
+#include "netmon_stats.h"
+
+LOG_MODULE_REGISTER(sntp_client, CONFIG_LOG_DEFAULT_LEVEL);
+
+/* Store elapsed real time since the system booted in milliseconds. */
+static int64_t time_base_ms;
+
+/* Get the timestamp in seconds since 1st Jan 1970 and store it
+ * in the time_base tracking variable.
+ */
+int sntp_client_get_time(time_t *sec)
+{
+	int ret;
+	struct sntp_time sntp_time;
+	// struct timespec time_spec;
+
+	LOG_INF("Sending SNTP request for current time:");
+
+	ret = sntp_simple(CONFIG_SNTP_INIT_SERVER, SYS_FOREVER_MS, &sntp_time);
+	if (ret < 0) {
+		LOG_ERR("Failed to SNTP current time: %d", ret);
+		return ret;
+	}
+
+	*sec = sntp_time.seconds;
+
+	// /* TODO:  Update the system clock time */
+	// time_spec.tv_sec = sntp_time.seconds;
+	// time_spec.tv_nsec = ((uint64_t)sntp_time.fraction * (1000 * 1000 * 1000)) >> 32;
+	// ret = clock_settime(CLOCK_REALTIME, &time_spec);
+	// if (ret != 0) {
+	// 	LOG_ERR("Failed to set system time, %d", ret);
+	// }
+
+    /* Increment the sntp time req stats counter */
```

Comment:
This seems to be indented by 4 spaces, not tabs.

Please make sure your editor is configured to use tabs for indentation, and to display tabs as 8 spaces, in order to get the right formatting for Zephyr's coding style.

Reply:
Just use clang-format and the config file in the root of Zephyr. If you're on VSCode you can install the clang-format plugin, and for your local project point formatting to that plugin, and reformat the whole file for Zephyr's style in one keyboard shortcut. CI uses the same file to verify formatting in PRs.